modules/charon-core/src/main/java/org/wso2/charon3/core/encoder/JSONEncoder.java

```diff
@@ -494,6 +494,17 @@ public String buildUserResourceTypeJsonBody() throws JSONException {
                     SCIMResourceSchemaManager.getInstance().getExtensionRequired());
             schemaExtensions.put(extensionSchemaObject);
 
+            if (Boolean.TRUE.equals(SCIMResourceSchemaManager.getInstance().isSystemUserExtensionEnabled())) {
```

Comment:
Remove the system extension enabled config and use a secial config only for the /resourceType api